Sources/SKTestSupport/Utils.swift

```diff
@@ -14,7 +14,7 @@
 package import Foundation
 package import LanguageServerProtocol
 import SwiftExtensions
-package import struct TSCBasic.AbsolutePath
+import struct TSCBasic.AbsolutePath
```

Comment:
Why not scope this import?

Reply:
It is scoped. We enable internal imports by default.

https://github.com/swiftlang/sourcekit-lsp/blob/bce09932ed87cb121b61bbbca1b3e970fa2c0d01/CMakeLists.txt#L15

https://github.com/swiftlang/sourcekit-lsp/blob/bce09932ed87cb121b61bbbca1b3e970fa2c0d01/Package.swift#L8